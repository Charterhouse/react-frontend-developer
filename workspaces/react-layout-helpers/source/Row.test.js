import React from 'react'
import { render } from 'react-testing-library'
import { Row } from './'

describe('Row', () => {
  it('has proper styling', () => {
    const { container } = render(<Row />)
    expect(container).toMatchSnapshot()
  })
})
