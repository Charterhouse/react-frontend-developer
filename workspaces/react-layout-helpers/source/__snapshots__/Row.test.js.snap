// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row has proper styling 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: row nowrap;
  -ms-flex-flow: row nowrap;
  flex-flow: row nowrap;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<div>
  <div
    class="emotion-0 emotion-1"
  />
</div>
`;
